from auth.auth import auth

auth()

... # dont need to check for auth(), if it fails it always quits ( i think )